doc/dothost.1

```diff
@@ -20,8 +20,21 @@ Show help message and exit.
 .TP
 .B \-\-version
 Show version information and exit.
-.SH EXAMPLE
+.SH EXAMPLES
 .EX
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as boxart > file.boxart"
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as html > file.html"
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as svg > file.svg"
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as dot > file.dot"
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as txt > file.txt"
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as vcg > file.vcg"
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as gdl > file.gdl"
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as graphml > file.graphml"
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as gif > file.gif"
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as jpg > file.jpg"
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as pdf > file.pdf"
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as png > file.png"
+.RB "$ " "dothost www.iana.org | graph-easy \-\-as ps > file.ps"
```

Comment:
In these formats, the images are saved as `graph.gif`, `graph.jpg` etc., not to stdout, so the redirects are incorrect.